lib/src/participant/local.dart

```diff
@@ -339,4 +339,18 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
   @internal
   Iterable<lk_rtc.TrackPublishedResponse> publishedTracksInfo() =>
       trackPublications.values.map((e) => e.toPBTrackPublishedResponse());
+
+  @internal
+  @override
+  bool setPermissions(ParticipantPermissions newValue) {
+    final didUpdate = super.setPermissions(newValue);
```

Comment:
In JS, we are passing back the [previous value](https://github.com/livekit/client-sdk-js/blob/main/src/room/events.ts#L233).. since you have an event.. perhaps we can pass back both?

Reply:
sure, like oldValue newValue ?

Reply:
since you have permissions, maybe just add `oldPermissions` ?

Reply:
got it